app/views/distributions/edit.html.erb

```diff
@@ -60,6 +60,7 @@
                   <%= f.input :issued_at, as: :datetime, ampm: true, minute_step: 15, label: "Distribution date" %>
                   <%= f.input :reminder_email_enabled, as: :boolean, checked_value: true, unchecked_value: false, label: "Send email reminder the day before?" %>
                   <%= f.input :agency_rep, label: "Agency representative" %>
+                  <%= f.input :delivery_method, collection: Distribution.delivery_methods.keys, label_method: :humanize, label: "Delivery method" %>
```

Comment:
As mentioned below, can we make this into radio buttons?

Reply:
done